### ISSUE TITLE
"npm run build-storybook" returns error "TypeError: Cannot convert undefined or null to object"

### ISSUE THREAD

**Issue by xxxKOTxxx** [emosheeep/fe-tools#15]
Faced same issue as described [here](https://stackoverflow.com/questions/77401779/npm-run-build-storybook-returns-error-typeerror-cannot-convert-undefined-or). 

Must to remove plugin until it will be fixed.

Thank You for attention.

**Comment by emosheeep** [comment]
The error stack picked from the link shows it was caused by `vite-plugin-dts/dist/index.mjs:581:100` 🤔, am I right? I'not sure, but it seems to be.

```
TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at configResolved (file://./node_modules/vite-plugin-dts/dist/index.mjs:581:100)
    at file://./node_modules/vite/dist/node/chunks/dep-bb8a8339.js:66031:28
    at Array.map (<anonymous>)
    at resolveConfig (file://./node_modules/vite/dist/node/chunks/dep-bb8a8339.js:66031:14)
    at async build (file://./node_modules/vite/dist/node/chunks/dep-bb8a8339.js:47926:20)
    at async build (./node_modules/@storybook/builder-vite/dist/index.js:159:11030)
    at async Promise.all (index 0)
    at async Module.build2 (./node_modules/@storybook/builder-vite/dist/index.js:159:13349)
    at async Promise.all (index 0)
```

**Comment by xxxKOTxxx** [comment]
I found that in my case error in DTS was causd by missing `build.lib` options from configeration.

When I copied my `build` options to `mergeConfig` in `main.ts` Storybook file, bundling also was successsful.

In any case after removing of `vite-plugin-lib-inject-css` DTS error gone.

I hope it help.

**Comment by emosheeep** [comment]
well，maybe these two plugins conflict on something. I'll take a look later when I'm free. a minimal reproduction will speed up it😉

**Comment by xxxKOTxxx** [comment]
I tried to remove DTS, but then next plugin fails as well :)

I can't share lib that I'm working on because of privacy reasons, but I suggest to use [Nx](https://nx.dev/) to generate React and Vue3 UI libraries with Storybook and then just test Your lib on them. Fails only `storybook-build` mode.

I hope it will be easily reproduced.

**Comment by emosheeep** [comment]
thanks a lot🙏

**Comment by emosheeep** [comment]
I succeed to reproduce it and know why it breaks, it not serious, I plan to fix it at v2.0 and publish in some days.

**Comment by emosheeep** [comment]
2.0.0-beta.0 is already published, you can have a try.
